### ISSUE TITLE
Can't access content with reverse proxy?

### ISSUE THREAD

**Issue by joshnat** [porn-vault/porn-vault#1422]
1. Running local install porn-vault on port 3000 and using Nginx Proxy Manager (docker) to redirect to the host.  
2. From within my LAN/WLAN I can log into the app with the IP:port of the machine it's running on and see all clips content/import activity.  
3. When using a reverse proxy that directs to the same IP:port, the UI appears, but there is no clips content.  UI has message "There was an error" and button for "Try Again" which doesn't have an effect.  
4. Password works in both cases
5. I assume the UI is not able to connect to Elasticsearch?
6. I experienced the same problem running porn-vault and Elasticsearch in Docker containers, which is why I tried installing them locally on the same machine.

**Desktop (please complete the following information):**
 - OS: Ubuntu 20.04
 - Browser: Chrome 
 - Version 90.0.4430.85

**Comment by leadwolf** [comment]
Are there any errors in the pv console ? If not, go to your browser, press f12, go to the network tab, try to load a page. The network tab should show a "ql" request. Click on it and show the response

**Comment by joshnat** [comment]
I figured most of it out.  ql was returning a 503 in the console.  I'm using Cloudflare for DNS, and had the Cloudflare proxy on for this subdomain.  When I turned off the Cloudfare proxy, ql was able to accept the query.  

Looking at the 503 error, it's Cloudflare DDOS protection getting in the middle and asking to turn on javascript before the query can be executed.  I'm not sure how to get around it, but at least I know the reason.
